back/engines/commercial/analytics/spec/factories/dimension_referrer_types.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :dimension_referrer_type_website, class: 'Analytics::DimensionReferrerType' do
```

Comment:
We could shorten a bit the name of dimension factories. For instance:
```suggestion
  factory :dim_referrer_type_website, class: 'Analytics::DimensionReferrerType' do
```

Reply:
We could also have a more generic one where `key` and `name` are generated by a sequence. This would allow us to create multiple channels at once:
```ruby
create_list(:dim_referrer_type, 3)
```